### PR TITLE
More wreck edits

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Edited some of the loot to be more consistent, less seed crates and crates that have single item in them.
Upped artifact spawner a tiny bit, and made a change that wont allow the arti to trigger on wreck anymore.

## Why / Balance
Arti trigger and wreck and send half the loot into deep space.

## How to test
N/A

## Media
N/A

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
:cl:
- tweak: Blue wrecks should now have more interesting loot.

### CODE REVIEW

**Comment by whatston3** [comment]
Tested as of 723f0c7 - everything seems sane.  Newest additions seem reasonable, let me know when this is finished.

File: Resources/Prototypes/_NF/Catalog/Fills/Crates/salvage.yml [hunk 1]
```diff
@@ -4,308 +4,60 @@
...
+          range: 0, 3
+
+- type: entityTable
+  id: NFSalvageAssortedGoodies
+  table: !type:GroupSelector
+    children:
+    # Mats
     - id: SheetPlasma
-      prob: 0.1
+      weight: 0.1
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Crates/salvage.yml:20]: Might recommend adding steel and glass if only to dilute this table somewhat, - it's _very valuable_ as-is, very compressable, and wrecks are _littered_ with these.

Also, the range selector is giving us 4 items - there's a 1 added to a random.NextFloat call, so you get a number between [0,4) then iterate from 0 _to_ that number - pick 3.91, iterate 0, 1, 2, 3 - four items.

**Review by whatston3 — APPROVED** [review]
Pushed a change - addressed the numberselector range issue, added some glass and steel to the salvage crates (0.15 weight ~~total~~ each, overall weight is now ~1.1-1.2).

Enormous wreck value is solidly about 55-70k.

Behaves well, wardrobe lockers are interesting.  Good overall changes.

**Comment by dvir001** [comment]
Might wanna lower overall worth later with more space trash but so be it.

**Pull request merged**